### PR TITLE
B2B-1139: Add types for onReceive, onError callbacks

### PR DESCRIPTION
**Pull request by scott-collins-mesh**

### CODE REVIEW

File: packages/front-b2b-link-rn/src/Front.tsx
Comment (balint-front) [inline packages/front-b2b-link-rn/src/Front.tsx:1]: Can you please remove `// @ts-nocheck` comment from the top of the file?

File: packages/front-b2b-link-rn/src/Types.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+export type BrokerType =
```
Comment (AlexZeml-Front) [inline packages/front-b2b-link-rn/src/Types.ts:1]: We'll have to keep this type consistent with one in front-web-platform as we add/modify it there, can we consider having it shared between repos or it's not a good idea ?

Comment (scottcollins-front) [reply packages/front-b2b-link-rn/src/Types.ts:1]: First attempt was to include directly from [JS SDK](https://github.com/FrontFin/front-b2b-link/blob/main/packages/link/src/utils/types.ts), but @balint-front was rightfully concerned about bloating the project with dependencies. Maybe we should have a small public repo for resources shared between front-web-platform and the various SDKs?

Comment (AlexZeml-Front) [reply packages/front-b2b-link-rn/src/Types.ts:1]: agree

**Review by alex-zemlyanukhin-mesh — APPROVED** [review]

File: packages/front-b2b-link-rn/src/Front.tsx
Comment (scottcollins-front) [reply packages/front-b2b-link-rn/src/Front.tsx:1]: Updated, thanks!

File: packages/front-b2b-link-rn/src/Types.ts (hunk 1 above)
Comment (balint-front) [reply packages/front-b2b-link-rn/src/Types.ts:1]: I think that should be the end goal. But I suggest to move it after the release.

Comment (front-eric) [reply packages/front-b2b-link-rn/src/Types.ts:1]: Yes, we can move it after the release.

**Pull request merged**